### ISSUE TITLE
rework std deviation view

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/infographiq#19]
Rather than showing limits for std dev, areas of the lines could be color-ized or highlighted where they go outside of a threshold. @mceachronfwc noted in our meeting that this would be a more elegant (but admittedly more complicated) approach.

Are we happy with the current display or do we want to re-work it?

**Comment by 7yl4r** [comment]
Suggested by Jenn Brown:

![image](https://user-images.githubusercontent.com/1051390/31025867-c72942c4-a533-11e7-831e-de9abe4ae5cb.png)

Re-work the colorization, not focusing on the last 5 yrs, instead focusing on the usual/unusual areas. 

Also: don't use green/red b/c of potential colorblindness.

**Comment by jbosch-noaa** [comment]
@7yl4r  - I like the top plot.  I think it emphasizes the anomaly better.

**Comment by bbest** [comment]
I agree with @jbosch-noaa. The horizontal shading between the +/-1 sd makes the anomalies pop out with white background .

The original vertical shading for the last 5 years and horizontal lines was intended to mimic the IEA, but for a more scientific audience:

![image](https://user-images.githubusercontent.com/2837257/31027703-0a811878-a500-11e7-9dde-f286bfc864b7.png)
